ansible/ACI/aci_create.yml

```diff
@@ -0,0 +1,23 @@
+# Description
+# ===========
+# This playbook create an Azure Container Registry.
```

Comment:
the file name is acr_create.yml right?

Reply:
No. It is aci_create.yml. The naming convention is <scenarioname>_<action>. This is for the ACI scenario.